opendevin/runtime/client/runtime.py

```diff
@@ -208,53 +292,65 @@ def init_sandbox_plugins(self, plugins: list[PluginRequirement]) -> None:
         pass
 
 
-    
 def test_run_command():
-    sid = "test"
+    sid = 'test'
     cli_session = 'main' + ('_' + sid if sid else '')
     event_stream = EventStream(cli_session)
     runtime = EventStreamRuntime(event_stream)
-    asyncio.run(runtime._run_command(CmdRunAction('ls -l')))
+    asyncio.run(runtime.run_action(CmdRunAction('ls -l')))
+
 
 async def test_event_stream():
-    sid = "test"
+    sid = 'test'
     cli_session = 'main' + ('_' + sid if sid else '')
     event_stream = EventStream(cli_session)
-    runtime = EventStreamRuntime(event_stream)
+    runtime = EventStreamRuntime(
+        event_stream,
+        sid,
+        'ubuntu:22.04',
```

Comment:
Recommend to test it in sandbox image to make sure the python execution enviroment the same.

Reply:
Sounds good! Will try to add some unit tests for it!

Reply:
Yeah, in my previous PR I generally test it in sandbox image rather than in `ubuntu`. So maybe this is the reason I meet the problem and introduce `self.clean_up` but you don't. 🤔 We don't need too everything in this PR. The unit test can be done in the future. :)